src/secure-proxy-handler.ts

```diff
@@ -206,12 +201,9 @@ export class SecureProxyHandler implements ProxyHandler<SecureProxyTarget> {
         this.initialize(shadowTarget);
         return key in shadowTarget;
     }
-    ownKeys(shadowTarget: SecureShadowTarget): (string | symbol)[] {
+    ownKeys(shadowTarget: SecureShadowTarget): PropertyKey[] {
         this.initialize(shadowTarget);
-        return push(
-            getOwnPropertyNames(shadowTarget),
-            getOwnPropertySymbols(shadowTarget)
-        );
+        return ownKeys(shadowTarget);
```

Comment:
We use `ownKeys` instead of `getOwnPropertyNames()` and `getOwnPropertySymbols()` since they're equiv.